File: app/src/main/java/com/example/pract27/ActivityClasses/MyBaseActivity.java
package com.example.pract27.ActivityClasses;

import androidx.appcompat.app.AppCompatActivity;

public class MyBaseActivity extends AppCompatActivity {
    protected static final int CREATE_ACTION = 0x000312;
    protected static final int EDIT_ACTION = 0x000313;
    protected static final String EXTRA_ID = "id";
    protected static final String EXTRA_ACTION_CODE = "actionCode";
    protected static final String EXTRA_NOTE = "note";
    protected static final String EXTRA_TIME = "time";
    protected static final String DATE_PATTERN = "dd.MM.yyyy HH:mm";
}

File: app/src/main/java/com/example/pract27/MyBaseActivity.kt
package com.example.pract27

import androidx.appcompat.app.AppCompatActivity

open class MyBaseActivity : AppCompatActivity() {
    companion object {
        protected val list = mutableListOf<String>()
        protected const val CREATE_ACTION = 0x000312
        protected const val EDIT_ACTION = 0x000313
        protected const val EXTRA_TEXT = "text"
        protected const val EXTRA_ID = "id"
        protected const val EXTRA_ACTION_CODE = "actionCode"
    }
}